packages/teleport/src/services/audit/audit.test.ts

```diff
@@ -75,6 +75,75 @@ test('fetch events', async () => {
         user: 'root',
       },
     },
+    {
+      codeDesc: 'Database Created',
```

Comment:
sorry i overlooked this, can we revert this file? this is just to test correct formatting of events when fetching with just select random events

Reply:
@kimlisa Isn't this a unit test for events formatting? I can revert but I thought it's useful to make sure your event message is formatted correctly for example. There's also a storybook test which I added them to as you asked, but that one is very messy with all the html.

Reply:
🤷‍♂️ I think it's much more convenient to test your event message formatting with this unit test than updating the storybook but it's your land :) I'll remove.